src/cluster/distributed-state-registry.js

```diff
@@ -98,6 +111,20 @@ module.exports = class DistributedStateRegistry extends EventEmitter{
 	}
 
 	/**
+	 * Returns all the servers with a shared interest
+	 *
+	 * @public
+	 * @returns {Array} entries
+	 */
+	getAllServers( name ) {
```

Comment:
Hold on, there's a bug here and consecutively in `src\utils\subscription-registry.js` on line 42.  `this._data[ name ].nodes` is a Object with
```
{
    serverName: <Boolean>
}
```
which allows for quicker lookups (no need for `indexOf`) and also multiple sets to the same state.